app/facades/food_facade.rb

```diff
@@ -0,0 +1,11 @@
+class FoodFacade
+  def self.total_items_returned(food)
+    FoodService.food_search(food)[:totalHits]
+  end
+
+  def self.details(food)
+    FoodService.food_search(food)[:foods].map do |food_data|
+      SearchedFood.new(food_data)
+    end
+  end
```

Comment:
Going off of the comment from before, you could do something like: 

```ruby 
  def self.details(food)
   search_data =  FoodService.food_search(food)[:foods]
   foods = search_data.map do |food_data|
      SearchedFood.new(food_data)
    end
   [foods, search_data[:total_hits]]
  end
```

Reply:
And then you'd be able to get rid of the other method (`total_items_returned`)

Reply:
We also want to limit the amount of foods that are being displayed to 10. You could do this either by limiting it before creating food objects (wouldn't want to limit it after creating ALL the food objects when you are then only going to take the first 10 anyway), OR you could find a way to limit what you get back from the api (check to see if the api allowed for a parameter such as `limit`).